src/main/kotlin/no/skatteetaten/aurora/boober/mapper/v1/AuroraVolumeMapperV1.kt

```diff
@@ -28,20 +28,11 @@ class AuroraVolumeMapperV1(private val applicationFiles: List<AuroraConfigFile>)
     }
 
     private fun createSecretVaultHandlers(): List<AuroraConfigFieldHandler> {
-        val secretVaultSubKeys = applicationFiles.findSubKeys("secretVault")
-
-        //no subkeys means this is the old format. Return fieldHandler for "secretVault"
-        if (secretVaultSubKeys.isEmpty()) {
-            return listOf(
-                    AuroraConfigFieldHandler("secretVault")
-            )
-        }
-
-        return secretVaultSubKeys.flatMap { secretVaultSubKey ->
-            listOf(
-                    AuroraConfigFieldHandler("secretVault/$secretVaultSubKey")
-            )
-        }
+        return listOf(
```

Comment:
I am not sure why I let this slip in the original PR. Sorry.

Reply:
not much of a slip, but it is clearer now